src/control/lib/ipmctl/nvm_ctest.go

```diff
@@ -7,7 +7,6 @@
 package ipmctl
 
```

Comment:
This file should only be built if nvm_defaultarch.go is also being built, so should have the same build constraints.

Reply:
Ok, Thanks.